modules/learn/pages/buckets-memory-and-storage/memory.adoc

```diff
@@ -9,31 +9,33 @@
+The memory quota you allocate for a service applies to every instance of that service across your cluster. 
+For example, if you allocate 2048 MB to the xref:services-and-indexes/services/analytics-service.adoc[Analytics Service], and you run the Analytics Service on three of a cluster's five nodes, each instance of the service has 2048 MB of memory.
 
-As a guideline, Couchbase recommends that no more than 90% of a node's memory (80% on nodes with a small amount of total memory) be allocated to the server and its services. 
-The firm limits on server memory allocation are: stem:[max(total\_memory - 1, 80\% \times total\_memory)] where `total_memory` is the maximum memory on the node in GiB.
+NOTE: You can't allocate different amounts of memory for different instances of the same service in a cluster. 
 
-Instructions on how to allocate memory quotas to services when initializing a new cluster can be found in the section xref:manage:manage-nodes/create-cluster.adoc[Create a Cluster].
+Couchbase recommends that you allocate no more than 90% of a node's memory (80% on nodes with a small amount of total memory) to a server and its services. 
 
-When a node is added to a cluster, the _Default Configuration_, as established by the set-up of the first node in the cluster, is available: this covers all configurable elements, including memory quotas.
-However, if insufficient memory for the default configuration is available on the new node, the default configuration is prohibited: in such cases, settings for the new node can be custom-configured, allowing an appropriate subset of services to be specified.
+The firm limits for server memory allocation can be calculated by: stem:[max(total\_memory - 1, 80\% \times total\_memory)] where `total_memory` is the maximum memory on the node in GiB.
 
-If, when the initial node of a cluster is set up, only a subset of services is assigned, additional nodes can subsequently be added, in order to host additional services: in which case, each new service can be given its initial memory allocation as its node is added.
+The xref:services-and-indexes/services/data-service.adoc[Data Service] must run on at least one node in any cluster. Every xref:buckets.adoc[bucket] you create on a node has its own memory quota. 
+The available memory for a bucket comes from the quota you assign to the Data Service. 
+For more information on bucket memory quotas, see the <<bucket-memory,Bucket Memory Quotas section>>.
 
-Instructions on how to add a new node to a cluster can be found in the section xref:manage:manage-nodes/create-cluster.adoc[Create a Cluster].
+When you add a new node, you can choose to use the same configuration from the first node in the cluster. 
+If this default configuration exceeds the memory available for the node, you can customize the new node's settings and change its assigned services. 
+You can add more nodes to a cluster to add more services. 
```

Comment:
This is true, but some administrators deliberately set up different services on different nodes. So when they add a new node, they will as you say "customize the new node's settings and change its assigned services", but it won't be because "the default configuration exceeds the memory available for the node"; it's because they _don't want to use_ the default configuration on the new node. Need to rewrite to reflect this.